src/helpers/apiError.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiErrorType } from '../typings/schema';
+
+const messagesByErrorType: Record<ApiErrorType, string> = {
+  MustNotBeAuthorizedError:
+    'This operation requires that the request is not authenticated',
+  MustBeAuthorizedError:
+    'This operation requires that the request is authenticated',
+  InvalidAccessTokenError:
+    'The passed access token is either empty, expired or invalid',
+};
+
+type ApiErrorOptions = {
+  isSensitive: boolean;
+};
+
+const defaultOptions: ApiErrorOptions = {
+  isSensitive: true,
```

Comment:
Any error thrown anywhere in a resolver call stack will be propagated up the call chain and handled by GraphQL. By default, all properties of the error object are displayed in the `errors` field of the GraphQL query. This deletes the error stack to prevent leaking private data. We may also need to hide more types of data later on.